### ISSUE TITLE
Modes tools-vm and snapshot-vm return UNKNOWN on hosts containing no VMs

### ISSUE THREAD

**Issue by sbraz** [centreon/centreon-vmware#77]
Running this command on a host containing no VMs
 ```
perl centreon_plugins.pl --plugin apps::vmware::connector::plugin --custommode connector --mode tools-vm --connector-hostname 127.0.0.1 --container='xxx'
```
results in:
```
UNKNOWN: Cannot find 'VirtualMachine' object |
```

**Comment by garnier-quentin** [comment]
What did you expect ? :)

**Comment by sbraz** [comment]
There's nothing wrong with having an empty ESX at times, perhaps there could be a flag that makes this OK? Something like `--no-vms-ok`?
I'll agree that this is no big deal though :)

**Comment by garnier-quentin** [comment]
It can be an issue if there is a filter. So i will add an option for that.

**Comment by garnier-quentin** [comment]
Available in version 3.2.2
